functions/indicatorScoreCalculation/lowIncome.R

```diff
@@ -0,0 +1,55 @@
+# 
+# data <- allData
+# geometry <- geometryFiles[[1]]
+# name <- names(geometryFiles)[[1]]
+
+processLowIncome <- function(geometry, name, data){
+  # select the data set of interest 
+  vals <- data[[grep(pattern = name, x = names(data))]] |> as.data.frame()
+  
+  # structure then generate and select measures of concern
+  output <- structureACS(vals) |>
+    dplyr::mutate(
+      percent_lowincome = ifelse(C17002_001 == 0, NA, (C17002_001 - C17002_008) / C17002_001))|>
```

Comment:
Curious what variable C17002_008 represents?

Reply:
There is a export of the ACS label values in data/products/acsLabels.csv

The ACS code was provide by CDPHE during the development of 1.0, so I haven't changed it much. 

the census table is ratio of income to poverty level in the past 12 months
001 is the total pop 
008 is the pop with an income of 2x poverty level